src/metabase/db/liquibase.clj

```diff
@@ -242,30 +303,24 @@
           (doseq [[^ChangeSet change-set fail-on-error?] fail-on-errors]
             (.setFailOnError change-set fail-on-error?)))))))
 
-(defn- changelog-table-name
-  "Returns case-sensitive database-specific name for Liquibase changelog table for db-type"
-  [db-type]
-  (if (#{:h2 :mysql} db-type)
-    "DATABASECHANGELOG"
-    "databasechangelog"))
 
-(s/defn consolidate-liquibase-changesets!
+(mu/defn consolidate-liquibase-changesets!
   "Consolidate all previous DB migrations so they come from single file.
```

Comment:
It's not a single file anymore, is it?

Reply:
updated https://github.com/metabase/metabase/pull/34400/commits/99ca116e8203d269e506c52db741913711d6516e